targets/targets.json

```diff
@@ -2740,6 +2740,58 @@
         ],
         "device_name": "STM32G071RBTx",
         "bootloader_supported": true
+    },
+	"NUCLEO_G031K8": {
+        "inherits": [
+            "MCU_STM32_BAREMETAL"
+        ],
+        "supported_form_factors": [
+            "ARDUINO",
+            "MORPHO"
+        ],
+        "core": "Cortex-M0+",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (need HW patch) | USE_PLL_HSI",
+                "value": "USE_PLL_HSI",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "lpticker_lptim": {
+                "help": "This target supports LPTIM. Set value 1 to use LPTIM for LPTICKER, or 0 to use RTC wakeup timer",
+                "value": 1
+            },
+            "hse_value": {
+                "help": "HSE default value is 25MHz in HAL",
+                "value": "8000000",
+                "macro_name": "HSE_VALUE"
+            }
+        },
+        "extra_labels_add": [
+            "STM32G0",
+            "STM32G031xx",
+            "STM32G031K8"
+        ],
+        "components_add": [
+            "FLASHIAP"
+        ],
+        "macros_add": [
+            "STM32G031xx",
+            "STM32G031K8",
```

Comment:
Need a STM32G031K8 macro ?

Reply:
I dont see it used in this PR , `STM32G031xx` should be sufficient?

Reply:
Resolved with 53d3a3f79ca0c691eafc4496034c9ef92b6a4ce9